tests/example-project-configs/aws/aws-github-fs.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
What's the purpose of this file? We don't run tests using this right?

Reply:
It's just the file I used to generate the template to test whether the changes were working. It's the same as the other one, except it generates an FS model.

Reply:
Hmm we don't really get much utility out of it except specify FS as yes, which if we did for each option, end up with tens of example configs. I think we should just remove this file then